netlify-cms-academic/static/admin/config.yml

```diff
@@ -165,6 +165,7 @@ collections:
       - {label: "Subtitle", name: "subtitle", widget: "string", required: false}
       - {label: "Body", name: "body", widget: "markdown"}
       - {label: "Publish this page on", name: "date", widget: "datetime"}
+      - {label: "Last modified", name: "lastmod", widget: "datetime", required: false, default: false}
```

Comment:
Please revert this `lastmod` change as we get the last modified date from Git and we're trying to keep the CMS UI fairly simple and minimal for now.

Reply:
Reverted.